src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Represents a <code>Storage</code> object to handle saving and loading.
+ */
+
+
+public class Storage {
+    private String filepath;
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Saves current tasklist as a file in specified filepath.
+     * @param tasklist
+     */
+    public void save(TaskList tasklist) {
+        try {
+            List<Task> ls = tasklist.getList();
+            File f = new File(filepath);
+            f.delete();
+            f.createNewFile();
+            FileWriter fw = new FileWriter(filepath);
+            for (Task t : ls) {
+                fw.write(t.saveString() + "\n");
+            }
+            fw.close();
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+    }
+
+    /**
+     * Loads contents of current saved file into tasklist.
+     * @return The list containing the pre-saved tasks.
```

Comment:
I suggest having a line between the description and the parmeter section, as written [here](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=Empty%20line%20between%20description%20and%20parameter%20section):D I noticed the same issue in other places too.